src/form/FileUpload/FileUploadStyles.ts

```diff
@@ -16,9 +16,14 @@ export const FileUploadStyle = styled.div`
 	cursor: pointer;
 `;
 
+export const FileUploadPlaceholderWrapper = styled.div`
+	display: flex;
```

Comment:
No direction or wrapping?

Reply:
default flex direction, it is meant for a line of text of which a part is hidden at a certain resolution